modules/nodes-about-bare-metal-hosts-and-nodes.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-about-bare-metal-hosts-and-nodes_{context}"]
+= About bare metal hosts and nodes
+
+In order to provision a bare metal host as a node in your cluster, you must first create a `machineset` CR object that corresponds to the bare metal host hardware. Bare metal host machine sets describe infrastructure components specific to your configuration. You apply specific Kubernetes labels to these machine sets and then update the infrastructure components to run on only those machines.
```

Comment:
@jtomasek can you check this section ? Isnt there some overlap with https://deploy-preview-30656--osdocs.netlify.app/openshift-enterprise/latest/machine_management/user_infra/adding-bare-metal-compute-user-infra.html ?

Reply:
@rawagner @jtomasek @andybraren I guess the detail that is missing here is some description of the requirements for a host to become a node on the cluster, ie, that the host must be provisioned with RHCOS, etc. I can add this and a link to the section you link to above.